### ISSUE TITLE
Scrum #2

### ISSUE THREAD

**Issue by truzopus** [crismihailescu/socialMotivator#21]
This past 2 weeks, I implemented the cardview component for the project using Material UI, such that clicking on a card in the carousal(by Cris) would give an enhanced view. Difficulties included understanding what others were trying to do so that mines would synchronize well with them, so I made the component as general as possible. This week, I'll start off by finishing up the sign up/sign in dialog pop up and implemented redux into most of our components. I will be researching react-hook-forms as an external library to help validate the sign up/sign in for repeated names.

r7k2b

**Comment by GaussianSums** [comment]
Over the last period, I implemented a very simple nav bar, and added some material ui components including a modal sign-up component and a modal popup. Over this period I will be adding functionality to the components I had worked on previously, as well as adding materialUI makeStyle styling, converting from the current CSS. I will also be learning about redux!

f0m1b

**Comment by crismihailescu** [comment]
The past two weeks I helped organize a lot of the tickets for enhancements, created the carousel component, and integrated the work Annie, Truman and myself did around the event cards. These next two weeks I will be working on the styling to make the webpage look for complete, as well as filling out the homepage with unique carousel components that are relevant to what the user wants to see. The main challenge from last week that is carrying over is working with Redux, which is quite complicated to understand at first, and hopefully as I start using it more it will begin to make sense. 

u2n1b

**Comment by meinna** [comment]
This iteration I worked on components on the home page of our website. Specifically, I worked on implementing the functionality of filtering by activity data type as well as a few smaller components. I found the hardest part of this iteration was completing a component without knowing completely how it should interact with other components as well as figuring out how to pass information from an inner component back up to an outer component. This coming iteration I plan to learn more about the useEffect hook and implement it in our project and convert previous generic css into material ui.

r7e9
